sites/public/.env.template

```diff
@@ -18,7 +18,7 @@ NEW_RELIC_LICENSE_KEY=
 GTM_KEY=GTM-KF22FJP
 
 ### Begin Doorway-specific variables ###
-# Note: This must be kept in sync with the same env variable in backend/core
```

Comment:
This variable is still in backend/core so can you keep both comments instead of deleting this one?

Reply:
I just added it back for the time being.  That var on the backend side is only temporary, so it will be removed prior to handoff.